Script/AtomicEditor/ui/frames/WelcomeFrame.ts

```diff
@@ -50,6 +50,20 @@ class WelcomeFrame extends ScriptWidget {
 
     addExample(example: ProjectTemplates.ProjectTemplateDefinition) {
 
+        var fileSystem = Atomic.getFileSystem();
```

Comment:
This doesn't need to be changed for this PR, though for reference there is a shortcut to the fileSystem on the Atomic object: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Script/TypeScript/AtomicWork.d.ts#L20

On another note, these should be a getters, or some other fancy TS construct :)